src/NationalInstruments.Analyzers/NationalInstruments.Analyzers.csproj

```diff
@@ -5,6 +5,7 @@
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <AssemblyName>NationalInstruments.Analyzers</AssemblyName>
     <RootNamespace>NationalInstruments.Analyzers</RootNamespace>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
```

Comment:
What does this attribute do?

Reply:
This attribute enables additional analyzer rules.  There's actually a warning if you don't enable this.